.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```

Comment:
- [ ] I think we should only check the php versions which are defined as version constraint in the composer.json. Therefore the php 7.2 check will always fail because the composer install task will crash. Can you remove the version?

Reply:
The development dependencies are no longer compatible with PHP 7.1.

Since the goal is to no longer support that specific PHP version, it does not make sense to spent further time to run the tests with that version.